cmd/sops/common/common.go

```diff
@@ -0,0 +1,84 @@
+package common
```

Comment:
This had to be moved to a different package to prevent import cycles. I'd want this to be in `cmd/sops`, but it can't be, because for example `cmd/sops/subcommand/edit` uses this, and `cmd/sops` uses `cmd/sops/subcommand/edit`.

Reply:
This package contains functions meant to be used by the `cmd/sops` binary. This package does not have a stable API guarantee. Because some subcommands are split under `subcommand/` packages, these functions can't be in `cmd/sops` because of import cycles. For instance, `cmd/sops/subcommand/groups` uses `LoadEncryptedFile`, and `cmd/sops` imports `cmd/sops/subcommand/groups`.

Reply:
should they be under `sops/cmd/internal` ?